cpp/src/lib/osd_info.cc

```diff
@@ -83,13 +91,19 @@ std::ostream &operator<<(std::ostream &os, const OsdInfo &info) {
 }
 
 std::ostream &operator<<(std::ostream &os, const OsdCapabilities &caps) {
-  os << "OsdCapabilities( port= ";
+  os << "OsdCapabilities( rora_port= ";
+
+  if (boost::none == caps.rora_port) {
```

Comment:
add an `operator<<` for `boost::optional` to `stuff.h` ? (alternative: there's something available in boost as well, if you add the right `#include` but it doesn't show it's an optional anymore)